draft-ion.adoc

```diff
@@ -105,16 +120,13 @@ A single Ion instance resource without metadata is simply a standard JSON object
 }
 ----
 
-A single Ion instance resource with metadata is a JSON object that has a direct
-child `meta` name/value pair in addition to its other member name/value pairs:
+An Ion _Resource_ is a JSON object with a Metadata member named `meta` in addition to one or more other members (name/value pairs):
```

Comment:
It makes sense if I understand your frame of reference. However, I am wrestling with the value add of these "semantic" concepts vs. just the semantics of the media-type. I am wondering what my value is to recognize something is a link, resource or collection resource vs. just understand the media-type itself and how to make decisions. 

E.g., 

If there is meta and an href, I can resolve that when I see it. Who cares what type of thing it is.

When if run across an ion object (has meta), then what information (data) does it have and what controls. What can I do with those controls. Is the data another ion object, what can I do with that. Why do I care if it is a resource vs. a link with the differentiator is it has 1+ members? I just care if there are data or controls there and what they mean.

IMO, adding these things unnecessarily bloats the spec and unnecessarily constrains a client. A good ion client should just allow you to introspect the message and know what you can do vs. categorize things that fine.

I think this spec is at heart really simple:

1. A meta object and what it means.
2. An Ion object and what it means.

Nothing more other than the constraint that it is at the root an object type. Everything else is overloading or adding concepts that don't need to be there.

If you put on the "data format" not "media-type" hat, what does a resource mean. An ion object becomes a resource when it is served as response to a request. 

Hope this is coming across the correct way, but I think things can be simpler here in a positive way.